text/logs/0130-logs-1.0ga-definition.md

```diff
@@ -0,0 +1,57 @@
+# Logs 1.0 GA Scope
+
+This document defines what's in scope for OpenTelemetry Logs 1.0 GA.
+
+## Motivation
+
+Clearly defined scope is important to align all logs contributors and to make
+sure we know what target we are working towards. Note that some of the listed
+items are already fully or partially implemented but are still listed for
+completeness.
+
+## Logs 1.0 GA Todo List
+
+### Specification and SDK
+
+- Write guidelines and specification for logging libraries to support
+  OpenTelemetry-compliant logs.
+
+- Implement OpenTelemetry logs SDK for Java that support trace context
+  extraction from the current execution context.
```

Comment:
Was this discussed with the maintainers of the Java project?

Reply:
> I had the understanding the Logs portion of OpenTelemetry was explicitly not a part of the upcoming GA. Has that changed?

@MrAlias GA here means "whenever we decide to GA logs support", not the upcoming GA where only trace and metrics will be supported. I should probably remove the reference to 1.0 to avoid confusion.

Reply:
Not yet, this is the opening of the discussion :-)
We have a logs SIG meeting tomorrow, Java maintainers can join to discuss, or log people can join the next Java SIG, whatever works best.

Reply:
I'm completely swamped right now, so if logs people could join the Java SIG, that would be best. My main concern is distracting/eating time from from the rest of the GA effort, and who will maintain the submitted log-oriented code in the meantime.

Reply:
I definitely don't want to distract Java SIG. We do not need to make detailed decisions now. The purpose of this PR is primarily to align people who already already are working on logs and create clarity around the goals. We will not be asking current Java SIG members to do any actual work for logs until traces and metrics are fully done and ready for GA.

However, if we can agree on the goals for logs I hope you guys don't mind if the alpha implementation of the logs is done in parallel by other contributors and we don't stall it for several months.

I'll join one of the coming Java SIG meetings to discuss this. @zenmoto would be great if you can join too. Let's sync.

Reply:
We've already been getting PRs from @zenmoto in the Java repo that is distracting me from other work, so yes, it would be great if y'all could come to the Java SIG and talk about the plan. Thanks!

Reply:
Unfortunately I will be unable to make this week's Java SIG, but I will plan to join next week (and I'll talk with @tigrannajaryan afterwards).

Reply:
Next week is fine.